.github/workflows/binary-combine-android.yml

```diff
@@ -0,0 +1,51 @@
+name: Android Build Combine
+
+on:
+  workflow_call:
+   inputs:
+      os:
```

Comment:
Why do we need to pass the OS here? We can just hardcode it to use ubuntu always.

Reply:
Done